apex-app-archetype/src/main/resources/archetype-resources/src/test/resources/log4j.properties

```diff
@@ -1,4 +1,4 @@
-log4j.rootLogger=DEBUG,CONSOLE
+log4j.rootLogger=INFO,CONSOLE
```

Comment:
In this case should not it be log4j.rootLogger=ALL,CONSOLE? What if user decides to use trace or own custom level, not debug? Using DEBUG level sounds related to the level used by com.datatorrent loggers.

I'll make the change and exclude archetype resource from the commit.